lte/protos/pipelined.proto

```diff
@@ -87,6 +84,12 @@ message ActivateFlowsRequest {
 
   // APN AMBR info, passed at the time of session creation
   magma.lte.AggregatedMaximumBitrate apn_ambr = 6;
+
+  // Subscriber session ipv4 address
+  string ip_addr = 2;
```

Comment:
shall we change the name to ipv4_addr to make it clear?

Reply:
I thought it's best to minimize changes if our end goal is to remove the ip address from this call

Reply:
sounds good.